src/privatesend-client.cpp

```diff
@@ -689,9 +700,13 @@ bool CPrivateSendClient::DoAutomaticDenominating(CConnman& connman, bool fDryRun
 {
     if(fMasternodeMode) return false; // no client-side mixing on masternodes
     if(!fEnablePrivateSend) return false;
-    if(!pwalletMain || pwalletMain->IsLocked(true)) return false;
```

Comment:
Condition `pwalletMain->IsLocked(true)` is not checked in new code. Is it OK?

Reply:
Ooops. This is ok but it's from another fix. Will revert here.